layout.link

```diff
@@ -220,22 +220,10 @@ WRAM0
 	org $c000
 	"Stack"
 	align 8
-	"Audio RAM"
-	"WRAM 0"
-	org $c308
```

Comment:
Don't these exist to minimize WRAM shifting, which breaks saves and needs darsh to update the save patch?

Reply:
Will have to look into that, was not aware anything from either of those sections were saved

Reply:
wOptions, wPlayerData, wPokemonData, wCurMapData, seem to be what gets saved and all of those besides wOptions reside in wramx

My debug save seems to load fine

Reply:
org directives have been there since linker scripts were first added in d9f530f8. can't find any explicit reasoning

`org $c308` is from 91f0530e

Reply:
To close this point, no, this doesn't affect saves. As ariscop said, everything I care about is in wramx.